library/core/src/cmp.rs

```diff
@@ -29,7 +29,7 @@ use self::Ordering::*;
 ///
 /// This trait allows for partial equality, for types that do not have a full
 /// equivalence relation. For example, in floating point numbers `NaN != NaN`,
-/// so floating point types implement `PartialEq` but not [`Eq`].
+/// so floating point types implement `PartialEq` but not [`trait@Eq`].
```

Comment:
This would show the text `trait@Eq` verbatim in the visible documentation.

You can keep just the text `Eq` in the visible text, but still link it to `trait@Eq`, like this:
```suggestion
/// so floating point types implement `PartialEq` but not [`Eq`][trait@Eq].
```

Reply:
Rustdoc actually special cases this to remove the disambiguator, so you can keep it as is if you like. Making it a reference link might be easier to read in code though.

Reply:
Oh sorry, rustdoc is again smarter than I'm assuming. (:

Reply:
Thanks for the review!  I did rebuild the docs to verify that syntax works.